notification/app/notification/controllers/Main.scala

```diff
@@ -34,7 +34,7 @@ final class Main(
 
   override def isPermittedTopic(apiKey: String): Topic => Boolean = {
     if (configuration.electionRestrictedApiKeys.contains(apiKey)) {
-      topic => topic.`type` == ElectionResults
+      topic => List(ElectionResults, LiveNotification).contains(topic.`type`)
```

Comment:
Should election restricted api keys (never heard of them but the name seems self explanatory) also be allowed to use this new topic type ?